web/src/markdown.js

```diff
@@ -388,7 +388,26 @@ function make_emoji_span(codepoint, title, alt_text) {
 }
 
 function handleUnicodeEmoji({unicode_emoji, get_emoji_name}) {
-    const codepoint = unicode_emoji.codePointAt(0).toString(16);
+    // We want to avoid turning things like arrows (↔️) and keycaps (numbers
```

Comment:
Nit: You’ve written that ↔️ qualified with an emoji variation selector in the comment, but we’re really talking about unqualified arrows.

```suggestion
    // We want to avoid turning things like arrows (↔) and keycaps (numbers
```

Reply:
oh ha, I was trying for the unqualified one and my text editor made it look like that oops